en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-api-manager.md

```diff
@@ -0,0 +1,320 @@
+      
+            The profiles supported in WSO2 API Manager 4.0.0 onwards for a distributed setup are only the Gateway Profile, Control Plane Profile, and the Traffic Manager Profile where the Key Manager component is embedded within the Control Plane. However, if your requirement is to continue using Opaque (Reference) tokens, you will need to maintain a separate Key Manager node or a Cluster to be able to scale it with the Gateway nodes. For instructions on how to configure a separate Key Manager profile along with the Control Plane, Gateway and Traffic Manager profiles, see [Deploying WSO2 API Manager in a Distributed Setup with Key Manager Separated]({{base_path}}/install-and-setup/setup/distributed-deployment/deploying-wso2-api-m-in-a-distributed-setup-with-km-separated).
+
+
+??? note "API Manager 3.1.0"
+
+    - API-M from 3.1.0 onwards supports only pdf, doc and txt type files as file typed API documents by default. If you wish to support other document types, please add the below config to `deployment.toml`.
+
+        ```tab="Format"
+        [apim.publisher]
+        supported_document_types = "<DOCUMENT TYPES LIST>"
+        ``` 
+
+        ```tab="Example"
+        [apim.publisher]
+        supported_document_types = "pdf, txt, zip, json, yaml"
+        ``` 
+
+    - From API-M 3.1.0 onwards, tag-wise grouping has been deprecated. Users are informed to use [API category based grouping](https://apim.docs.wso2.com/en/latest/reference/customize-product/customizations/customizing-the-developer-portal/customize-api-listing/api-category-based-grouping/) instead.
+
+    - New user store managers with inbuilt unique ID support was introduced in WSO2 5.10.0 and named with the UniqueID prefix. User store managers that do not have UniqueID as part of the user store manager name are only available for backward compatibility purposes and can only be used if you are migrating from a previous version of WSO2 Identity Server.
+  
+    Please check [Migration of user store managers with unique ID support](https://is.docs.wso2.com/en/5.11.0/setup/migrating-what-has-changed/#migration-of-user-store-managers-with-unique-id-support) for more information.
+
+??? note "API Manager 3.0.0"
+
+    - If you have used OIDC or SAML2 for SSO in the portal login in an older APIM version, you have to re-configure them as per the [documentation to configure WSO2 IS as an external IDP using OIDC]({{base_path}}/install-and-setup/setup/sso/configuring-identity-server-as-external-idp-using-oidc/).
+
+    - Until WSO2 API Manager 2.6.0, users had to update multiple configuration files to configure the product. This overhead is removed with the new configuration model because now users only have to update a single file (deployment.toml).
```

Comment:
```suggestion
    - Until WSO2 API Manager 2.6.0, users had to update multiple configuration files to configure the product. This overhead is removed with the new configuration model because now users only have to update a single file (`deployment.toml`).
```